packages/devextreme/testing/tests/DevExpress.ui.widgets/splitter.tests.js

```diff
@@ -1335,6 +1335,49 @@ QUnit.module('Behavoir', moduleConfig, () => {
         assert.strictEqual($resizeHandle.css('width'), '5px');
     });
 
+    [
+        { orientation: 'horizontal', dimension: 'width' },
+        { orientation: 'vertical', dimension: 'height' },
+    ].forEach(({ orientation, dimension }) => {
+        [
+            { separatorSize: '50vh' }, { separatorSize: '20spx' }, { separatorSize: 'd10' }, { separatorSize: 'NaN' },
+            { separatorSize: '2%' }, { separatorSize: '20em' }, { separatorSize: '1vw' }, { separatorSize: '' }
+        ].forEach(({ separatorSize }) => {
+            QUnit.test(`Resize handle ${dimension} should fallback to default if separatorSize is incorrect on init (orientation=${orientation})`, function(assert) {
+                this.reinit({
+                    dataSource: [{ size: '500px' }, { size: '500px' }],
+                    orientation,
+                    separatorSize
+                });
+
+                this.assertLayout([50, 50]);
+
+                const $resizeHandle = this.getResizeHandles();
+
+                assert.strictEqual($resizeHandle.css(dimension), '8px');
+            });
+        });
+    });
+
+    [
```

Comment:
['horizontal', 'vertical'].forEach((orientation) => {
    ['', '', '',].forEach((separatorSize) => {
         QUnit.test(`Resize handle should fallback to default if separatorSize is incorrect on init, separatorSize=${sepratorSize}, orientation=${orientation}`) {
         }
    })
})

Value of dimension is autocalculated and relates from orientation value.